app/components/listing/LabelTagRows.tsx

```diff
@@ -3,44 +3,48 @@ import { LabelTag } from "components/ui/LabelTag";
 import { Service } from "../../models";
 import styles from "./LabelTagRows.module.scss";
 
-const LabelTagRows = ({ service }: { service: Service }) => {
-  const topLevelCategories = service.categories?.filter(
+const LabelTagRows = ({
+  categories,
+  eligibilities,
+}: {
+  categories?: Service["categories"];
+  eligibilities?: Service["eligibilities"];
+}) => {
```

Comment:
**note:** I changed the interface to accept only what the component needs. The "whole object pattern" is useful in some cases when there is need to work with most or all of the passed object. But in this case we just need a couple attributes of the `Service`. This also makes it much easier to mock data in the test rather than having to mock a VERY large `Service` object.